rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
```

Comment:
- [ ] The `:person` and `:book` should be writable also, kindly make sure to add them to `attr_accessor` so it is accessible (read/write) by the class